integrations/crewai/src/ucai_crewai/toolkit.py

```diff
@@ -0,0 +1,170 @@
+import json
+import logging
+from typing import Any, Callable, Dict, List, Optional
+
+from crewai_tools import BaseTool as CrewAIBaseTool
```

Comment:
Are we missing any dependencies? Installing `crewai>=0.67.1` then this line fails with `ModuleNotFoundError: No module named 'crewai_tools'`